### PR TITLE
change titles on pages

### PR DESCRIPTION
**Pull request by LBHABEBIC**
tiny PR updating titles on pages.

### CODE REVIEW

File: pages/people/[id]/records/index.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const AddNewRecordPage = (): React.ReactElement => {
   const personId = Number(query.id as string);
   return (
     <>
-      <Seo title={`#${query.id} Cases`} />
+      <Seo title={`#${query.id} New Record`} />
```
Comment (sirLisko) [inline pages/people/[id]/records/index.tsx:14]: `Add a new record to #id`

File: pages/people/[id]/allocations/add.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ const AddNewAllocationPage = (): React.ReactElement => {
   }
   return (
     <>
-      <Seo title={`#${query.id} Cases`} />
+      <Seo title={`#${query.id} Allocate Worker`} />
```
Comment (sirLisko) [inline pages/people/[id]/allocations/add.tsx:22]: what about `Allocate Worker to #`

File: pages/people/[id]/allocations/[allocationId]/remove.tsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ const RemovedAllocationPage = (): React.ReactElement => {
   }
   return (
     <>
-      <Seo title={`#${query.id} Cases`} />
+      <Seo title={`#${query.id} Deallocate Worker`} />
```
Comment (sirLisko) [inline pages/people/[id]/allocations/[allocationId]/remove.tsx:23]: `Deallocate Worker from #`

**Review by sirLisko — COMMENTED** [review]
What do you think? 🤔

**Comment by LBHABEBIC** [comment]
> What do you think? 🤔

yep makes sense

**Review by sirLisko — APPROVED** [review]
Perfect 👌

**Pull request merged**